lodash.src.js

```diff
@@ -3592,6 +3594,15 @@
         }
         return fn.apply(thisBinding, args);
       }
+
+      if (isAryExact) {
+        var plist = [];
+        for (var i = 0; i < ary; i++) { plist.push('p' + i); }
+        var params = plist.join(',');
+        var fn = wrapper;
+        eval('wrapper = function(' + params + ') { return fn.apply(this, arguments); }');
```

Comment:
@d10 true. I was not considering such environments. As it's an opt-in feature that only occurs when passing the third argument as true, `_.ary(fn, n, true)`, it seems that the impact to users in this environment would be small. I don't see much chance of them being confused if attempting to use this feature & having it fail either.

I believe this is the only way to handle arbitrary airity. To handle arity up to a fixed amount, one could create pre-defined wrappers for argument counts of 0-20, for instance, and fail for functions with more than 20 arguments. Only handling functions up to a certain length seems limiting, dangerous, and cumbersome.

Anyway, thanks for the observation. I still do believe this feature to be useful, but ultimately I will leave it up to whatever standard process you usually follow when considering adding features to the project.